### ISSUE TITLE
Kit / Easy-Assemble Build Availability

### ISSUE THREAD

**Issue by colinoflynn** [newaetech/chipshouter-picoemp#1]
One goal is to eventually offer a "partial kit", which will be in $50 to $100 range most likely. This kit would require you to solder the R-Pi Pico & some switches onto the board, but the SMD part would be **done**. This board would also be hi-pot + functionally tested,

*NOTE: A "full assembly" kit is probably never going to be offered, as the cost of kitting the SMD parts would be more than the cost of the SMD run. SMD kits have a higher failure rate which adds considerable "average support costs" ontop. Due to the availability of very low cost PCB services like JLCPCB or OSHPARK if you want a full kit please order the PCB & parts directly.*

**Comment by gmgunderground** [comment]
Well done, I will wait for it!

**Comment by SoftwareEngineer** [comment]
Nice! Very interested in a "partial kit". Will it be possible to put a header on the board instead of soldering the R-Pi pico right on the board?

**Comment by david-oswald** [comment]
Definitely interested in 1-2 of them.

**Comment by inode-** [comment]
Waiting for it!

**Comment by grymoire** [comment]
Put me down for a maybe.

**Comment by matthew5025** [comment]
+1 would be interested too!

**Comment by m-ober** [comment]
Count me in.

**Comment by jcldf** [comment]
yeah! I'm interested

**Comment by avloginov** [comment]
I'd be happy to buy either easy to assemble or just empty board.

**Comment by osok** [comment]
I am interested

**Comment by FlUxIuS** [comment]
Would be happy to test the easy-assemble build! 
Thanks in advance!

**Comment by doegox** [comment]
Yup, I'm interested by the easy-assemble kit too :)

**Comment by jonahedwards** [comment]
Definitely interested!

**Comment by RobinFassina-Moschini** [comment]
Interested by the easy-assemble kit too.

**Comment by KypForty** [comment]
Interested by mostly complete SMD board. Regards!

**Comment by jslawek** [comment]
![image](https://user-images.githubusercontent.com/10602939/149380169-0b61f2a8-594d-486e-b9e2-45075798eab9.png)

**Comment by gohai** [comment]
Interested in partial kit

**Comment by AlexanderKrampe** [comment]
Interested +1

**Comment by shinmai** [comment]
+1 for sure

**Comment by LovelyA72** [comment]
Sign me up!

**Comment by gworona** [comment]
Interested

**Comment by Gabbo28** [comment]
Interested!

**Comment by max-r-b** [comment]
Interested as well! Thanks!

**Comment by hexsecs** [comment]
Definitely interested!

**Comment by jamesy0ung** [comment]
+1

**Comment by Xenador77** [comment]
Interested as a southerner in a colloquialism

**Comment by hgordon-git** [comment]
super interested. I would get one to three.

**Comment by pr0z78** [comment]
Yes hyped and interested for at least 2 (shared buy for co-worker) or 3 (also 1 for team) when it  becomes available.

**Comment by afflitto** [comment]
very interested!

**Comment by zommiommy** [comment]
+1

**Comment by steelfly33** [comment]
+1

**Comment by colinoflynn** [comment]
Thanks for the patience all - we've released our first run of the kit at https://store.newae.com/chipshouter-picoemp . If you'll be at RECON in Montreal we have a special discount - free shipping if you pick up in person (use code RECON). Note the following:

* Requires 2xAA batteries for use (excluded as we weren't sure if it caused shipping issues, it looks like maybe not but we hadn't ordered them for the kit builds so we don't want to delay on that)
* We ship via DHL - we cannot ship via mail as historically the service is very unreliable & it's not worthwhile for us. This makes the shipping cost a large % of the unit cost - you can save a bit by splitting shipping, 4 fit in our smallest box.

We'll eventually see if there is easier ways to stock this - but for now in this initial run the logistics are likely going to be a bit heavy!

**Comment by autoprog** [comment]
Hi, 
in Moser not have https://store.newae.com/chipshouter-picoemp
Do you ship to Brazil?,
thanks,
Carlos.

**Comment by mrx23dot** [comment]
Any update on availability? What about selling on Tindie or Sparkfun, I think they fullfil orders.

**Comment by eried** [comment]
:) still no mouser? :/

**Comment by FreedomLabsIO** [comment]
> Thanks for the patience all - we've released our first run of the kit at https://store.newae.com/chipshouter-picoemp . If you'll be at RECON in Montreal we have a special discount - free shipping if you pick up in person (use code RECON). Note the following:
> 
> * Requires 2xAA batteries for use (excluded as we weren't sure if it caused shipping issues, it looks like maybe not but we hadn't ordered them for the kit builds so we don't want to delay on that)
> * We ship via DHL - we cannot ship via mail as historically the service is very unreliable & it's not worthwhile for us. This makes the shipping cost a large % of the unit cost - you can save a bit by splitting shipping, 4 fit in our smallest box.
> 
> We'll eventually see if there is easier ways to stock this - but for now in this initial run the logistics are likely going to be a bit heavy!

Hi @colinoflynn , the link is down. Are you going to make new batches available?

**Comment by mrx23dot** [comment]
Also no https on newae.com

**Comment by jpcrypt** [comment]
Our online store has been reorganized, find it here: https://www.newae.com/product-page/chipshouter-picoemp-kit
Currently out of stock but more batches will get made.
